packages/toml/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "@covector/toml",
+  "version": "0.0.0",
+  "files": [
+    "pkg/covector_toml_bg.wasm",
+    "pkg/covector_toml.js",
+    "pkg/covector_toml.d.ts",
+    "dist/index.js",
+    "dist/index.d.ts"
+  ],
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "scripts": {
+    "build": "wasm-pack build --release --no-pack --target nodejs && rimraf pkg/.gitignore",
+    "postbuild": "cp ./pkg/covector_toml_bg.wasm ../action/dist && cp ./pkg/covector_toml.js ../action/dist && npm run build:ts",
+    "build:debug": "wasm-pack build --dev --no-pack --target nodejs && rimraf pkg/.gitignore && npm run postbuild",
+    "build:ts": "tsc -b",
+    "clean": "rimraf dist tsconfig.tsbuildinfo node_modules",
+    "pretest": "npm run build",
```

Comment:
We do run a `build` in CI before running the tests, so I am wondering if we could possibly remove this. However, would we want this for first run locally with tests?

Reply:
I guess we could remove and just run it locally if needed